python/lsst/ts/mtaos/mtaos.py

```diff
@@ -305,6 +334,24 @@ async def configure(self, config: typing.Any) -> None:
         else:
             self.wep_config = dict()
 
+        if config.camera == "comcam":
+            self.camera = ComCam(
+                self.domain,
+                log=self.log,
+                intended_usage=ComCamUsages.MonitorState,
+            )
+            self.camera_name = "LSSTComCam"
+        elif config.camera == "lsstCam":
+            self.camera = LSSTCam(
+                self.domain,
+                log=self.log,
+                intended_usage=LSSTCamUsages.MonitorState,
+            )
+            self.camera_name = "LSSTCam"
+        self.oods = self.camera.rem.ccoods
```

Comment:
Just create a `salobj.Remote` for the OODS instead of creating and entire `LSSTCam` or `ComCam` object. However, I don't think you need a class attribute to hold this. You can probably do this inside the closed loop.